backend/tests/integration/alembic/test_alembic.py

```diff
@@ -200,9 +213,14 @@ def _get_table_names(db_alembic):
 
 
 def _get_custom_enums(db_alembic):
-    query = """ SELECT pg_type.typname AS enumtype,
-        pg_enum.enumlabel AS enumlabel
-        FROM pg_type
-        JOIN pg_enum
+    query = """ SELECT pg_type.typname AS enumtype,  
+        pg_enum.enumlabel AS enumlabel 
+        FROM pg_type  
+        JOIN pg_enum  
```

Comment:
you've added a bunch of trailing whitespace here.

Reply:
It's black :/

Reply:
Ok, it was my IDE, fixed.